producer:
    datamodel:
        ✔ data model @done(20-03-05 11:39)
            ✔ name as string @done(20-03-05 11:39)
            ✔ message_id as str from name and uuid.uuid4() @done(20-03-05 11:39)
            ✔ timestamp as str as datetime.utcnow() @done(20-03-05 11:39)
            ✔ lat as confloat  -90 < x < 90 @done(20-03-05 11:39)
            ✔ lon as confloat  -180 < x < 180 @done(20-03-05 11:39)
        ✔ test data model @done(20-03-05 11:39)

    endpoint:
        ☐ add /producer/<topicname> endpoint
            ☐ add aiokafka producer to handle incoming requests async and not block
        ☐ test /producer/<topicname> endpoint

consumer:
    ☐ response model



TIL

why do I need pykafka for aiokafka producer to connect?

It doesn't. The problem is that producer does not wait for Kafka to accept connections, so ConnectionError is thrown
on start-up. So by installing pykafka and restarting the producer-container I coincidally waited for kafka/zookeeper to
be up and running.
